mlflow/mleap.py

```diff
@@ -16,10 +16,12 @@
 import mlflow
 from mlflow.models import Model
 from mlflow.exceptions import MlflowException
+from mlflow.utils import keyword_only
 
 FLAVOR_NAME = "mleap"
 
 
+@keyword_only
```

Comment:
@dbczumar I think you mentioned these might be used in sagemaker container? i didn't see them used anywhere actually - let me know if I'm missing something (hopefully the tests will tell me if I am but just in case)

Reply:
`add_to_model` is used in `mlflow.spark.save_model`

Reply:
right, I wasn't going to keyword_only `add_to_model` since the `sample_df` param is the last one. i guess we might as well... :-/